kmod/scoutfs-kmod.spec.in

```diff
@@ -117,8 +111,8 @@ SCOUTFS_RPM_NAME=$(rpm -q %{name} | grep "%{version}-%{release}")
 rpm -ql $SCOUTFS_RPM_NAME | grep '\.ko$' > /var/run/%{name}-modules-%{version}-%{release} || true
 
 %postun
-if [ -x "%{weak_modules}" ]; then
-    cat /var/run/%{name}-modules-%{version}-%{release} | "%{weak_modules}" --remove-modules --no-initramfs
+if which weak-modules; then
```

Comment:
I think I'd write this myself as this:

```
if [ -x "$(which weak_modules)" ]; then
```

It would be cleaner and not print out `/usr/sbin/weak_modules` to the output.

Reply:
Sold!